shell/shared/stats.js

```diff
@@ -20,7 +20,11 @@ if (Meteor.isServer) {
   function computeStats(since) {
     return {
       activeUsers: Meteor.users.find({lastActive: {$gt: since}}).count() +
-          DeleteStats.find({type: "user", lastActive: {$gt: since}}).count(),
+          DeleteStats.find({$or: [{type: "user"},
```

Comment:
Instead of doing a possibly-expensive "or" query, maybe just add `appDemoUsers` into `activeUsers` after the fact?